data-and-functions-1/src/functions/getOrderInfo.js

```diff
@@ -0,0 +1,20 @@
+import getUserById from './getUserById'
+import getProductById from './getProductById'
+
+const getOrderInfo = (data) => {
+  if (data == null || data.orders == null) {
+    return null
+  } else {
+    const orders = []
```

Comment:
Since we've already got `data.orders`, and since the function is called `getOrderInto`, lets call this array `orderInfos` instead.